lib/stages/visAccumulate.cpp

```diff
@@ -304,6 +305,13 @@ void visAccumulate::main_thread() {
 
         // Start and end times of this frame
         timespec t_s = ((chimeMetadata*)in_buf->metadata[in_frame_id]->metadata)->gps_time;
+        // If GPS time is not set, fall back to the system's best guess at the time.
+        if (t_s.tv_sec == 0) {
```

Comment:
This is probably cleaner to test `Telescope::instance().gps_time_enabled()`, plus then you only need to do the test once and then just save the result.